insightconnect_plugin_runtime/api/endpoints.py

```diff
@@ -660,6 +668,15 @@ def get_plugin_info(plugin_spec_json, fields):
             plugin_info.update({field: plugin_spec_json.get(field)})
         return plugin_info
 
+    def get_plugin_sdk_version(self):
+        try:
+            version = pkg_resources.require("insightconnect-plugin-runtime")[0].version
+        except Exception as e:
```

Comment:
Would it be possible to change all instances of `e` to `error` to try and avoid single letter variables? 😃  
```suggestion
        except Exception as error:
```

Reply:
For now I'm going to leave this as-is as it's the standard try/except style used throughout the file and codebase, but in the future I would be on-board with a refactor to change them to `error`!